src/app/components/wallet-connect/wallet-connect.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="wallet-container">
+  <button class="btn" (click)="login()" *ngIf="!session">Connect Wallet</button>
+  <div *ngIf="session">
+    <p>Connected as: {{ session.actor }}</p>
+    <button class="btn" (click)="logout()">Disconnect Wallet</button>
+  </div>
+</div>
```

Comment:
```suggestion
<div class="wallet-container">
    <button class="btn" (click)="login()" *ngIf="!session">Connect Wallet</button>
    <div *ngIf="session">
        <p>Connected as: {{ session.actor }}</p>
        <button class="btn" (click)="logout()">Disconnect Wallet</button>
    </div>
</div>
```

Four-space indentation is preferred.